app/models/tasks/root_task.rb

```diff
@@ -77,7 +77,7 @@ def hide_from_task_snapshot
   end
 
   def available_actions(user)
```

Comment:
```suggestion
  # :reek:UtilityFunction
  def available_actions(user)
```

To hide the CC issue.